tests/JsonApiErrorMiddlewareTest.php

```diff
@@ -46,13 +56,17 @@ public function handle(ServerRequestInterface $request): ResponseInterface
         $response = $this->middleware->process($this->request, $nextHandler);
 
         $this->assertEquals(500, $response->getStatusCode());
-        $this->assertEquals('An exception occurred', $response->getReasonPhrase());
+        // TODO NOW: If no message was set then the Default error messages should be used!
+        $this->assertEquals('Internal Server Error', $response->getReasonPhrase());
         $this->assertJsonStringEqualsJsonFile(
             __DIR__ . '/expectations/exceptionWithoutCodeAndMessage.json',
             $response->getBody()->getContents()
         );
     }
 
+    // TODO NOW: Is the expected json-file really correct? Do we really want to have a text in there?
+    // TODO NOW: Mismatch of Internal Code and Http-Status-Code!!! Talk about this!!!
```

Comment:
Jap, true…

Reply:
We won't check the code (as the code can't be varified due to the fact that everyone can choose their own codes), but we will check the status.